test/e2e/autoscale_test.go

```diff
@@ -428,7 +428,7 @@ func TestTargetBurstCapacity(t *testing.T) {
 
 	// Now read the service endpoints and make sure there are 2 endpoints there.
 	// As always poll, since network programming takes times...
-	if err := wait.Poll(250*time.Millisecond, cfg.ScaleToZeroGracePeriod, func() (bool, error) {
+	if err := wait.Poll(250*time.Millisecond, cfg.StableWindow+cfg.TickInterval, func() (bool, error) {
```

Comment:
Network programming time has nothing to do with StableWindow and TickInterval. Adding a comment here to explain why this is set would be helpful for the next person to look a this.